components/coachindicator/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@spectrum-css/coachindicator",
+  "version": "1.0.0",
+  "description": "The Spectrum CSS coachindicator component",
+  "license": "Apache-2.0",
+  "author": "Adobe",
+  "homepage": "https://opensource.adobe.com/spectrum-css/coachindicator",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/adobe/spectrum-css.git",
+    "directory": "components/coachindicator"
+  },
+  "bugs": {
+    "url": "https://github.com/adobe/spectrum-css/issues"
+  },
+  "main": "dist/index-vars.css",
+  "scripts": {
+    "build": "gulp"
+  },
+  "peerDependencies": {
+    "@spectrum-css/button": ">=10",
+    "@spectrum-css/buttongroup": ">=5 <=6",
+    "@spectrum-css/tokens": ">=10"
+  },
+  "devDependencies": {
+    "@spectrum-css/button": "^10.1.3",
+    "@spectrum-css/buttongroup": "^6.0.50",
+    "@spectrum-css/component-builder": "^4.0.11",
```

Comment:
Should this one be using `component-builder-simple` here and in the `gulpfile.js`?

Reply:
hmmm this is a good question, this was started pre-DIY migrations so the goal was to just make the separate component but not migrate it yet. 

@pfulton Do you have a preference on if I should do the DIY Migration for Coach Indicator in this PR or as a separate one?

Reply:
Decided on adding the DIY migration for coach indicator into this PR